omero/developers/Web/CreateApp.rst

```diff
@@ -6,86 +6,93 @@ to get you familiar with the Django framework. The more you know about
 Django, the easier you will find it working with the OmeroWeb framework.
 One major feature of Django that we do not use in OmeroWeb is the Django
 database mapping, since all data comes from the OMERO server and is
-saved back there. You will notice that the models.py files in each app
-are empty.
+saved back there.
```

Comment:
I would reformulate the whole senetce 

``One major feature of Django that we do not use in OmeroWeb is the Django database mapping, since all data comes from the OMERO server and is saved back there.``

maybe simpler (the sentence is confusingly complex for no obvious benefit):

``We do not use Django database mapping in OmeroWeb, since all data comes from the OMERO server and is saved back there.``

Reply:
I think I can probably just remove the whole sentence.